products/blender.md

```diff
@@ -123,22 +127,11 @@ releases:
 > entirety of the 3D pipeline—modeling, rigging, animation, simulation, rendering, compositing and
 > motion tracking, video editing and 2D animation pipeline.
 
-## [Long Term Support](https://wiki.blender.org/wiki/Process/LTS)
-
-LTS releases are supported for 2 years with critical bug-fixes.
-
-For non-LTS releases, Blender only has a corrective release if severity 1 issues (high
-priority bugs) are found. When the corrective release is agreed on, however, severity 2 (high
-priority and normal bugs) fixes are ported along.
-
-For LTS releases, a more limited policy applies. Only severity 1 issues are ported (after the
-next stable release), on a fixed schedule (e.g., every 3 months) after the fix was tested in master
-for some time (e.g., 1 week).
-
-## [Release Schedule](https://code.blender.org/2023/02/blender-release-cycle-update/#new-release-schedule)
+[Since February 2023](https://code.blender.org/2023/02/blender-release-cycle-update/#new-release-schedule),
+Blender follows a 3-releases per year schedule, with one of the releases offering long-term support (Blender LTS).
 
-An updated release schedule was proposed in [February 2023](https://code.blender.org/2023/02/blender-release-cycle-update/#new-release-schedule),
-with the following goals:
+Non-LTS releases are supported until the next release with high priority bugs fixes. High severity and normal bugs
+may be fixed too if agreed on by the team.
 
-- A new LTS release every year around June, supported for two years,
-- A major version and breaking release every 2 years (3.0, 4.0, 5.0…).
+LTS releases are supported for 2 years. At first the support level is the same as non-LTS releases. But after the next
+release is out they are supported with critical bug-fixes only.
```

Comment:
Since these lines are being rewriting these lines, we might as well apply [semantic line-breaking](https://rhodesmill.org/brandon/2012/one-sentence-per-line/) to make reading the source (and future edits) easier:

```
Non-LTS releases are supported until the next release with high priority bugs fixes.
High severity and normal bugs may be fixed too if agreed on by the team.

LTS releases are supported for 2 years. At first the support level is the same as non-LTS releases.
But after the next release is out they are supported with critical bug-fixes only.
```

Reply:
So far the recommendation is to break at 100 characters, see https://github.com/endoflife-date/endoflife.date/wiki/Style-Guide#text.

Reply:
Fair enough. I might propose a change in a separate thread.

Reply:
Can you open a dedicated issue first, so the team can discuss about it ?

Reply:
Yeah, that's what I meant, sorry for the ambiguity :)

Reply:
Opened https://github.com/endoflife-date/endoflife.date/discussions/6975.